src/onegov/org/forms/event.py

```diff
@@ -505,7 +515,13 @@ def run_import(self):
         headers = self.headers
         session = self.request.session
         events = EventCollection(session)
-        tags = {self.request.translate(tag[0]): tag[0] for tag in TAGS}
+        all_tags = chain(
+            TAGS,
+            self.custom_tags() if self.custom_tags() else []
```

Comment:
`self.custom_tags() or []` would be even neater, you can use `or` short-circuiting to provide defaults for values that would evaluate to `False`.